dnf/cli/commands/group.py

```diff
@@ -132,16 +132,26 @@ def _list(self, userlist):
         uservisible = 1
         showinstalled = 0
         showavailable = 0
-        if len(userlist) > 0:
+        verbose = self.base.conf.verbose or self.opts.ids
+
+        while userlist:
             if userlist[0] == 'hidden':
```

Comment:
I'm not sure I understand this correctly and I didn't dig too deep... but the command line switches here don't have the `--` prefix (i.e. there is 'hidden', not '--hidden'), which means they can be mixed with the group names that come after them?

Maybe this has to do with some backwards compatibility too? Making "ids" another group name that can't be used seems bad and unnecessary (however improbable).

Reply:
This is yum compatibility where yum used keywords instead of options.